deken

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-# Ensure this file is executable via `chmod a+x deken`, then place it
+# Ensure this file is executable via $(chmod a+x deken), then place it
```

Comment:
To my mind using backticks is a way of telling the user they should execute a command. That was my intention here.